HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_03.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    rasult_array = time_line(total_rows(log))
+    array_of_duration(rasult_array).size == 1 ? array_of_duration(rasult_array)[0] : array_of_duration(rasult_array)
```

Comment:
you use this method 3 times with the same argument, would be better to add memoization for it or save result to local variable